llvm/test/tools/llvm-readobj/ELF/symbols.test

```diff
@@ -13,9 +13,11 @@
 # RUN: yaml2obj %s -DBITS=64 -DTYPE=ET_REL -o %t64
 # RUN: llvm-readobj --symbols %t64 | FileCheck %s --match-full-lines --strict-whitespace --check-prefix=SYMBOLS-LLVM
 # RUN: llvm-readelf --symbols %t64 | FileCheck %s --match-full-lines --strict-whitespace --check-prefix=SYMBOLS-GNU64
+# RUN: llvm-readelf -s --extra-sym-info %t64 | FileCheck %s --match-full-lines --strict-whitespace --check-prefix=SYMBOLS-GNU64X
```

Comment:
Is it worth adding a check showing that --extra-sym-info has no impact on the output for LLVM output format? That way it's a clear indication where any testing belongs, should the behaviour change in the future.

Reply:
Added